src/wallet/test/fuzz/crypter.cpp

```diff
@@ -27,36 +27,36 @@ FUZZ_TARGET(crypter, .init = initialize_crypter)
     // These values are regularly updated within `CallOneOf`
     std::vector<unsigned char> cipher_text_ed;
     CKeyingMaterial plain_text_ed;
-    const std::vector<unsigned char> random_key = ConsumeRandomLengthByteVector(fuzzed_data_provider);
+    const std::vector<unsigned char> random_key = ConsumeFixedLengthByteVector(fuzzed_data_provider, WALLET_CRYPTO_KEY_SIZE);
 
     LIMITED_WHILE(good_data && fuzzed_data_provider.ConsumeBool(), 10000)
     {
         CallOneOf(
             fuzzed_data_provider,
             [&] {
-                const std::string random_string = fuzzed_data_provider.ConsumeRandomLengthString();
+                const std::string random_string = fuzzed_data_provider.ConsumeRandomLengthString(100);
```

Comment:
Is there a specifc reason for 100 here or is it not that important?

Reply:
I used the size we reserve in the RPC for this as reference.